src/Pinion.ts

```diff
@@ -150,6 +165,7 @@ class Pinion {
     await this.ipfsNode.start();
     logDebug(`Pinner id: ${this.ipfsNode.id}`);
     await this.storeManager.start();
+    await this.announce();
```

Comment:
Why do we do this?

Reply:
This is to announce the presence of this pinner to a room that already has peers in it, so that they pick it up as soon as it's ready.